docs/source-lit/conf.py

```diff
@@ -398,13 +398,13 @@ def find_source():
 from typing import Optional
 
 import torch
-import pytorch_lightning as pl
+import lightning.pytorch as pl
```

Comment:
Why are we updating the docs import? The rest of the docs don't use the newer imports, no?

Reply:
this docs `source-lit` is specific as they will refer only to `lightning` package

Reply:
but more important with this package you won't get `lightning_app` nor `pytorch_lightning` anymore so in such case your package will be completely broken